spec/classes/repo_spec.rb

```diff
@@ -10,7 +10,14 @@
         is_expected.to contain_anchor('pulpcore::repo')
         is_expected.to contain_file('/etc/yum.repos.d/pulpcore.repo')
           .with_content(%r{^baseurl=https://yum.theforeman.org/pulpcore/\d+\.\d+/el\d+/\$basearch$})
-          .that_comes_before('Anchor[pulpcore::repo]')
+          .that_notifies('Anchor[pulpcore::repo]')
+        is_expected.to contain_package('pulpcore-dnf-module')
```

Comment:
this is currently expected on EL7 too, leading to failures:

```
pulpcore::repo on centos-7-x86_64 is expected to contain Package[pulpcore-dnf-module] that requires File[/etc/yum.repos.d/pulpcore.repo]
```

Reply:
Yes, trying to quickly do this before the next meeting is proving to be not the best strategy.